Dockerfile

```diff
@@ -1,6 +1,14 @@
 # Build the manager binary
 FROM quay.io/operator-framework/helm-operator:v1.6.1
 
+LABEL name="Sumologic Kubernetes Collection Helm Operator"
+      maintainer="support@sumologic.com"
+      vendor="sumologic"
+      version="0.0.0"
+      release="0"
```

Comment:
* version could be injected as build arg, WDYT?
* what's the purpose of release? incremented on every release? do we really need it? if so also a build arg?

Reply:
I second Patryk's question. Does the release come from the OpenShift?
What is `release` about?

Reply:
`release` label indicates release number and should be incremented on every release, @deekshahegde86 am I right?

Reply:
@deekshahegde86 Can labels in Dockerfile use variables defined in build args? I mean this form:
```
LABEL name="Sumologic Kubernetes Collection Helm Operator"
      maintainer="support@sumologic.com"
      vendor="sumologic"
      version=$VERSION
      release=$RELEASE_NUMBER
      summary="Sumologic Kubernetes Collection Helm Operator for the Sumo Logic Kubernetes Collection Helm Chart"
      description="Sumologic Kubernetes Collection Helm Operator deploys https://github.com/SumoLogic/sumologic-kubernetes-collection"
```
and then `$VERSION` and `$RELEASE_NUMBER` will be specified in build args e.g:
```
 docker build --build-arg VERSION="0.0.0"  --build-arg RELEASE_NUMBER="0" .
```

Reply:
@kkujawa-sumo  Yes you can.

Reply:
> `release` label indicates release number and should be incremented on every release

confirmed by @deekshahegde86